beacon-chain/rpc/eth/beacon/handlers_validator.go

```diff
@@ -49,7 +51,21 @@ func (s *Server) GetValidators(w http.ResponseWriter, r *http.Request) {
 	}
 	isFinalized := s.FinalizationFetcher.IsFinalized(ctx, blockRoot)
 
-	rawIds := r.URL.Query()["id"]
+	var rawIds []string
+	if r.Method == http.MethodGet {
+		rawIds = r.URL.Query()["id"]
+	} else {
+		err = json.NewDecoder(r.Body).Decode(&rawIds)
```

Comment:
I saw the spec and this list should be a unique list of IDs, i don't think decodeIds is checking for uniqueness.

Reply:
I don't see it anywhere in the spec. It is a useful thing to add, but I would prefer to have it in another PR.